.github/workflows/python-quality.yml

```diff
@@ -18,15 +18,20 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
-    - name: Set up Python
-      uses: actions/setup-python@v2
-      with:
-        python-version: 3.9
-    - name: Install dependencies
-      run: |
-        pip install --upgrade pip
-        pip install .[dev]
-    - run: black --check tests src
-    - run: isort --check-only tests src
-    - run: flake8 tests src
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+      - name: Install dependencies
+        run: |
+          pip install --upgrade pip
+          pip install .[dev]
+      - run: black --check tests src
+      - run: isort --check-only tests src
+      - run: flake8 tests src
+      - run: python utils/check_static_imports.py
```

Comment:
@sgugger @LysandreJik I just realized we are not using `make quality` in the CI. Any reason for that ?
If not, I think I'll replace it by the make command so that changes are reflected in the CI without changing the workflow again.

Reply:
I think we just copied from transformers, so it's for the same reason!

Reply:
Make sense :+1: